SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -221,12 +221,12 @@ - (void)run
     }
```

Comment:
It looks like you used some kind of auto formatting in this file. Could you undo it so that I can see what actually changed, and also so we can keep our diff from Signal here as minimal as possible?

Reply:
Right i did Xcode auto indent int he whole file ... I'll fix it up on monday